### PR TITLE
metadata export

### PR DESCRIPTION
**Pull request by mroz22**
image
 

- [x] export raw files to make them transferable between google drive and dropbox https://satoshilabs.slack.com/archives/CKN5C51CJ/p1731686820096629
- #7406 - possibly reformat data to match standard export/import format

### CODE REVIEW

**Comment by bosomt** [comment]
**QA OK**

- tested on 2 independent wallets and devices 
- exported from dropbox by mistake - lost of 429 errors from Dropbox, expected as export from dropbox is not needed/supported
- exported from Google, archive file was generated
- imported into Dropbox into Trezor directory
- After i loaded wallet account/wallet/transactions labels were present 

**Info:**
 - Suite version: desktop 24.12.0 (fe586851abda63dde5fa47366758545b9c73cd08)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/24.12.0 Chrome/130.0.6723.91 Electron/33.1.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.3 regular (revision 7f373ae71eca855dd41b4a0fdcc7cadaa13a8281)
 - Transport: BridgeTransport 3.0.0-bundled.24.12.0

**Comment by mroz22** [comment]
Thanks @bosomt. It was a blind shot by me so I am happy to see it works. The fact that it works makes it low-hanging fruits now. You can only add some toasts to handle the error branches and that's it. 

It can maybe already be useful for support? Also I was thinking that we maybe could add hidden routes so that support doesn't need to ask users to enable debug.

**Comment by karliatto** [comment]
@mroz22 I have been testing and it works, I just needed rebasing it before otherwise I was getting some errors not sure why.

I am having a look if some error handling in the UI is required or something.

But from observation and some testing I think this will open at least 2 follow-up tasks:

* We need to allow importing.
* I think in order to be compatible with https://github.com/bitcoin/bips/blob/master/bip-0329.mediawiki it requires to be in unencrypted .json so it can be import by other wallets as well.

**Comment by mroz22** [comment]
> I am having a look if some error handling in the UI is required or something.

I think I already forced pushed some notifications toasts to handle potential errors. So for purposes of debug mode, it should be ok. 

> We need to allow importing.

yes, next step.

> I think in order to be compatible with https://github.com/bitcoin/bips/blob/master/bip-0329.mediawiki it requires to be in unencrypted .json so it can be import by other wallets as well.

yes, next step.

File: packages/suite/src/views/settings/SettingsDebug/Metadata.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        dispatch(exportMetadataToLocalFile()).finally(() => {
+            setExporting(false);
+        });
+    };
+
+    return (
+        <SectionItem data-testid="@settings/debug/metadata">
+            <TextColumn
+                title="Export"
+                description="Export labeling files to your computer. You may use this to transfer your labeling files from your Google drive account to your Dropbox account."
```
Comment (karliatto) [inline packages/suite/src/views/settings/SettingsDebug/Metadata.tsx:24]: Nit: This is not translated but I guess for it is fine for debug menu

**Review by karliatto — APPROVED** [review]
Looks good to me!
I really like the idea of extending Suite export/import functionality. It could be very useful for users.

File: packages/suite/src/views/settings/SettingsDebug/Metadata.tsx (hunk 1 above)
Comment (mroz22) [reply packages/suite/src/views/settings/SettingsDebug/Metadata.tsx:24]: yeah that should be good for now.

**Pull request merged**